client/src/main/java/com/networknt/client/ssl/APINameChecker.java

```diff
@@ -0,0 +1,42 @@
+package com.networknt.client.ssl;
+
+import java.security.cert.CertificateException;
+import java.security.cert.X509Certificate;
+import java.util.Set;
+
+import javax.net.ssl.SSLException;
+
+import org.apache.http.conn.ssl.DefaultHostnameVerifier;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class APINameChecker {
+	private static final Logger logger = LoggerFactory.getLogger(APINameChecker.class);
+	private static final DefaultHostnameVerifier verifier = new DefaultHostnameVerifier();
```

Comment:
Could you add some comments to this class & methods to describe what it does & how it's being used?
Also would there be benefit to unit test these to ensure validation criteria is maintained in the future?

Reply:
ok. I'll add some unit tests. But this class is mostly a wrapper of the apache httpclient DefaultHostnameVerifier. Javadocs are available online. I'll add references to that class.